Dockerfile

```diff
@@ -23,7 +23,7 @@ EXPOSE 5050
 ENV GIT_HASH $GIT_HASH
 ENV HELM_DEBUG 1
 # WARNING: If you are changing any versions here, update it in the reference.conf
-ENV TERRA_APP_SETUP_VERSION 0.0.2
+ENV TERRA_APP_SETUP_VERSION 0.0.3
```

Comment:
Bumped setup chart back to 0.0.3 now that https://github.com/DataBiosphere/terra-app/pull/24 is merged (I repushed the chart, will verify GKE tests).